fusion-cli/build/babel-plugins/babel-plugin-i18n/index.js

```diff
@@ -45,6 +49,30 @@ function i18nPlugin(babel /*: Object */, {translationIds} /*: PluginOpts */) {
             }
             translationIds.add(element.value);
           });
+        } else if (specifierName === 'useTranslations') {
+          const localName = refPath.parentPath.parent.id.name;
```

Comment:
Might want to validate the parentPath type is what you expect. For example, someone could do this 

```js
function MyComponent() {
  const obj = {};
  obj.translate = useTranslations();
}
```
or

```js
function MyComponent() {
  const obj = {};
  return <OtherComponent translate={useTranslations()} />
}
```

Reply:
`useTranslations` can only be used as a CallExpression within a VariableDeclarator now